src/Webapi/Webapi__Dom/Webapi__Dom__Element.re

```diff
@@ -5,18 +5,13 @@ let ofNode = (node: Dom.node) : option('a) =>
     None;
 
 module Impl = (T: {type t;}) => {
-  let asHtmlElement: T.t => Js.null(Dom.htmlElement) = [%bs.raw
-    {|
-    function (element) {
-      // BEWARE: Assumes "contentEditable" uniquely identifies an HTMLELement
-      return element.contentEditable !== undefined ?  element : null;
+  let asHtmlElement: T.t => option(Dom.htmlElement) = [%raw {|
+    function(element) {
+      if (element instanceof HTMLElement) return element;
```

Comment:
This won't work for elements created in a different window (e.g. an iframe). The old code wasn't great, but it at least passed that case. Here's how we do it in TinyMCE, using `isPrototypeOf` instead of `instanceof`:
https://github.com/tinymce/tinymce/blob/develop/modules/sand/src/main/ts/ephox/sand/api/HTMLElement.ts#L15-L20

Reply:
Fixed in https://github.com/reasonml-community/bs-webapi-incubator/blob/4f9a3624f47571d0474b2e2e6de5f7ab0bea340a/src/Webapi/Webapi__Dom/Webapi__Dom__Element.re#L8 , thanks for pointing this out!

Reply:
Thanks for pointing this out, should be fixed in https://github.com/reasonml-community/bs-webapi-incubator/blob/08e00d32c280ad6de8209114e825a56f842cbaa3/src/Webapi/Webapi__Dom/Webapi__Dom__Element.re#L9

Reply:
Cool. That's pretty much how the TinyMCE code works, thanks to the `resolve` helper function, I can't remember why we stuck to `isPrototypeOf` instead of `instanceof` for the final check but it should be the same.